Composer/packages/extensions/visual-designer/src/utils/jsonTracker.ts

```diff
@@ -152,18 +152,22 @@ export function deleteNodes(inputDialog, nodeIds: string[], callbackOnRemovedDat
 }
 
 export function insert(inputDialog, path, position, $type) {
-  const dialog = cloneDeep(inputDialog);
-  const current = get(dialog, path, []);
   const newStep = {
     $type,
     ...seedNewDialog($type, { name: generateSDKTitle({ $type }) }),
```

Comment:
❓Should we move the name generation into `seedNewDialog`?

Reply:
I think so, we need a cleanup on all usages of seedNewDialog. Let me handle that in another PR.
![image](https://user-images.githubusercontent.com/8528761/76478826-d443fa80-6444-11ea-9f04-fe4e062febcf.png)
![image](https://user-images.githubusercontent.com/8528761/76478839-e2921680-6444-11ea-8e8c-a2216a78ab99.png)
@a-b-r-o-w-n do you think it's necessary to rename `seedNewDialog` to `seedAdaptiveElement` or some other names?

Reply:
https://github.com/microsoft/BotFramework-Composer/issues/2244

Reply:
Thanks @yeze322